cci_nepal/getters/dummy_data/get_dummy_data.py

```diff
@@ -0,0 +1,13 @@
+# Import libraries
+import pandas as pd
+
+
+# Read Excel file
+def read_excel_file(file_path):
```

Comment:
I think the idea is to have specific read ins of data for specific tasks. So steering away from just read_excel and read_csv - see here: https://nestauk.github.io/ds-cookiecutter/structure/#fetchingloading-data-srcgetters

And here (kuebiko/getters/sic.py* ): https://nestauk.github.io/kuebiko/episodes/episode_01/